### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'group'

### ISSUE THREAD

**Issue by sergiomora03** [ssut/py-googletrans#248]
### This error comes from the issue #48  . Which was closed but they did not solve.

**Googletrans version:**
- [ ] **3.x**
- [x] **2.x**

**I'm submitting a ...** 

- [x] bug report
- [ ] feature request

**Current behavior:**
AttributeError: 'NoneType' object has no attribute 'group'

**Expected behavior:**
At least translate: 'Hola Mundo'

**Steps to reproduce:**
Just try translate any sentence...

**Related code:**
```
from googletrans import Translator
translator = Translator()
translator.translate('Hola Mundo')
```

**Other information:**
The complete error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-f71bcb40be3a> in <module>
----> 1 translator.translate('Hola Mundo')

~\Anaconda3\lib\site-packages\googletrans\client.py in translate(self, text, dest, src)
    170 
    171         origin = text
--> 172         data = self._translate(text, dest, src)
    173 
    174         # this code will be updated when the format is changed.

~\Anaconda3\lib\site-packages\googletrans\client.py in _translate(self, text, dest, src)
     73             text = text.decode('utf-8')
     74 
---> 75         token = self.token_acquirer.do(text)
     76         params = utils.build_params(query=text, src=src, dest=dest,
     77                                     token=token)

~\Anaconda3\lib\site-packages\googletrans\gtoken.py in do(self, text)
    184 
    185     def do(self, text):
--> 186         self._update()
    187         tk = self.acquire(text)
    188         return tk

~\Anaconda3\lib\site-packages\googletrans\gtoken.py in _update(self)
     63 
     64         # this will be the same as python code after stripping out a reserved word 'var'
---> 65         code = unicode(self.RE_TKK.search(r.text).group(1)).replace('var ', '')
     66         # unescape special ascii characters such like a \x3d(=)
     67         if PY3:  # pragma: no cover

AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by WilliamVoong** [comment]
This issue have already been opened at issue #234

**Comment by ssut** [comment]
I've released [googletrans@3.1.0a0](https://pypi.org/project/googletrans/3.1.0a0/) on PyPI and it seems to be working. Note that you all have to specify the package version like: `pip install googletrans==3.1.0a0`.
